routes/story.js

```diff
@@ -1,39 +1,61 @@
 const express = require("express");
 const router = express.Router();
+const Story = require('../models/Story');
 
 // @desc Story summary
 // @route GET /story/:storyId
 // @access Public
 router.get("/:storyId", (req, res) => {
-  res.send("fetch story details with that params.storyId");
+    const { storyId } = req.params.storyId;
+    try {
+        Story.findOne({ _id: storyId }, (err, story) => {
+            if (!story) throw "Invalid Story ID";
```

Comment:
Right now you are just throwing an error in case of story not being found but what when we find a story? 
Try using await to get the story data and send the result as a response.

Reply:
Changes incorporated